controllers/csi/provisioner/reconciler.go

```diff
@@ -101,14 +101,14 @@ func (r *OneAgentProvisioner) Reconcile(ctx context.Context, request reconcile.R
 		return reconcile.Result{}, fmt.Errorf("failed to fetch connection info: %w", err)
 	}
 
-	envDir := filepath.Join(r.opts.DataDir, ci.TenantUUID)
+	envDir := filepath.Join(r.opts.RootDir, dtcsi.DataPath, ci.TenantUUID)
 	verFile := filepath.Join(envDir, "version")
-	tenantFile := filepath.Join(r.opts.DataDir, fmt.Sprintf("tenant-%s", dk.Name))
+	tenantFile := filepath.Join(r.opts.RootDir, dtcsi.DataPath, fmt.Sprintf("tenant-%s", dk.Name))
```

Comment:
what happens with multiple tenants and same dynakube name?

Reply:
We can't have multiple tenants within one dynakube instance

Reply:
what about same name (not instance) and different namespace?